distributed/comm/tests/test_ucx.py

```diff
@@ -264,42 +258,35 @@ def test_ucx_deserialize():
     assert result["op"] == "ping"
 
 
-@pytest.mark.skip(reason="hangs")
-@pytest.mark.parametrize("processes", [True, False])
-def test_ucx_localcluster(loop, processes):
-    if processes:
-        kwargs = {"env": {"UCX_MEMTYPE_CACHE": "n"}}
-    else:
-        kwargs = {}
-
+def test_ucx_localcluster(loop):
     ucx_addr = ucp.get_address()
     with LocalCluster(
         protocol="ucx",
-        interface="ib0",
+        host=HOST,
         dashboard_address=None,
         n_workers=2,
         threads_per_worker=1,
-        processes=processes,
+        processes=False,
         loop=loop,
-        **kwargs
     ) as cluster:
         with Client(cluster) as client:
             x = client.submit(inc, 1)
             x.result()
             assert x.key in cluster.scheduler.tasks
-            if not processes:
-                assert any(w.data == {x.key: 2} for w in cluster.workers.values())
+            assert any(w.data == {x.key: 2} for w in cluster.workers.values())
             assert len(cluster.scheduler.workers) == 2
 
 
+@pytest.mark.skip(reason="TCP not supported")
 def test_tcp_localcluster(loop):
     ucx_addr = "127.0.0.1"
     port = 13337
     env = {"UCX_MEMTYPE_CACHE": "n"}
     with LocalCluster(
-        2,
+        n_workers=2,
+        protocol="ucx://",
```

Comment:
Note, you may not have intended this protocol change

Reply:
Fixed